src/app/pages/template/stripe-template/stripe-template.component.ts

```diff
@@ -0,0 +1,168 @@
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { Router } from '@angular/router';
+import { Stripe, StripeCardElement, StripeElements } from '@stripe/stripe-js';
+// Lazy loading of stripe script
+import { loadStripe } from '@stripe/stripe-js/pure';
+import { ActionResponse } from 'types/DataResult';
+import { User } from 'types/User';
+
+import { CentralServerService } from '../../../services/central-server.service';
+import { ComponentService } from '../../../services/component.service';
+import { MessageService } from '../../../services/message.service';
+import { SpinnerService } from '../../../services/spinner.service';
+import { HTTPError } from '../../../types/HTTPError';
+import { BillingSettings, BillingSettingsType } from '../../../types/Setting';
+import TenantComponents from '../../../types/TenantComponents';
+import { Utils } from '../../../utils/Utils';
+
+//declare var Stripe: any;
+
+@Component({
+  selector: 'app-stripe-template',
+  templateUrl: './stripe-template.component.html',
+  styleUrls: ['./stripe-template.component.scss']
+})
+
+export class StripeTemplateComponent implements OnInit {
+
+  // Billing & Billing Information
+  public isActive = false;
+  public billingSettings!: BillingSettings;
+
+  @ViewChild('cardInfo', { static: true }) cardInfo: ElementRef;
```

Comment:
Needed?

Reply:
The whole line
`@ViewChild('cardInfo', { static: true }) cardInfo: ElementRef;`